GoogleForCreators/web-stories-wp
# Karma: run on CI

## Summary

Closes #1721.

~Blocked by #1809 for the Percy part.~

## Relevant Technical Choices

Does not include coverage reporting just yet.

Makes files compatible with current Node v14 setup.

## To-do

ℹ️ For follow-up PR:

* [ ] Actually make code coverage working   
    * Right now no coverage file is generated. But there's at least a summary in the console.
    * Is `babel-plugin-istanbul` needed?

## User-facing changes

N/A

## Testing Instructions

Verify that GitHub actions workflow here is running successfully.

---



See #1721
